src/Microsoft.OpenApi.Readers/V2/OpenApiResponseDeserializer.cs

```diff
@@ -37,7 +39,7 @@ internal static partial class OpenApiV2Deserializer
             {
                 "schema", (o, n) =>
                 {
-                    n.Context.SetTempStorage("operationschema", LoadSchema(n));
+                    _responseSchema = LoadSchema(n);
```

Comment:
Why this change to a static?  This is going to make the deserializer not thread safe.  That is most likely going to be a problem.

Reply:
You are right. I wasn't thinking about that issue. I'll revert this back. There are a few instances like this in the code, so I'll fix those separately.

---
In reply to: [165093793](https://github.com/Microsoft/OpenAPI.NET/pull/197#discussion_r165093793) [](ancestors = 165093793)